etl/steps/data/garden/hyde/2024-01-03/all_indicators.py

```diff
@@ -0,0 +1,76 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("all_indicators")
+    # Read table from meadow dataset.
+    tb = ds_meadow["all_indicators"].reset_index()
+
+    # Load meadow dataset.
+    ds_codes = paths.load_dataset("general_files")
+    tb_codes = ds_codes["general_files"].reset_index()
+
+    #
+    # Process data.
+    #
+    # Add country name
+    tb = tb.rename(columns={"country": "iso_code"}).astype({"iso_code": "str"})
+    tb_codes = tb_codes.astype("str")
+    tb = tb.merge(tb_codes, on="iso_code", how="left")
+    tb.loc[tb["iso_code"] == "Total", "country"] = "World"
+    # Drop columns
+    tb = tb.drop(columns=["iso_code"])
```

Comment:
Add `errors="raise"`.

Reply:
Argument `errors` is set to `"raise"` for both `drop` and `merge` by default.